src/httpserver.cpp

```diff
@@ -320,7 +320,7 @@ static bool HTTPBindAddresses(struct evhttp* http)
 
     // Bind addresses
     for (std::vector<std::pair<std::string, uint16_t> >::iterator i = endpoints.begin(); i != endpoints.end(); ++i) {
-        LogPrint(BCLog::HTTP, "Binding RPC on address %s port %i\n", i->first, i->second);
+        LogPrintf("Binding RPC on address %s port %i\n", i->first, i->second);
```

Comment:
```suggestion
        LogPrintLevel(BCLog::HTTP, BCLog::Level::Info, "Binding RPC on address %s port %i\n", i->first, i->second);
```

Reply:
Maybe I'm doing something wrong but this change with `./src/bitcoind -regtest -rpcbind=127.0.0.1 -rpcallowip=127.0.0.1/0` doesn't appear to display this log.

Reply:
me neither, it needs `-debug=http`

Reply:
Ah, I should have mentioned that the default logging levels were debug and trace so as not to change behavior until branch-off, but it's planned to be info from here on out, will update that now.

Reply:
Info levels will be unconditionally logged if 118c7567f62df2b88 is merged. For now, only warning and error messages are unconditionally logged. Feel free to leave it as LogPrint{f} for now and it will be updated in one of the #25203 steps.

Reply:
I'm going to leave this as is; think this makes sense to merge sooner rather than later.